scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/core/LiveJobsTest.java

```diff
@@ -147,6 +147,57 @@ public void testGetRunningTasksEmpty() {
         assertThat(liveJobs.getRunningTasks().isEmpty(), is(true));
     }
 
+    @Test(timeout = 600000)
+    public void testGetRunningTaskByTaskId() throws UnknownTaskException {
+        PASchedulerProperties.NUMBER_OF_EXECUTION_ON_FAILURE.updateProperty("5");
+
+        JobId id = new JobIdImpl(666L, "test-name");
+        TaskId tid = TaskIdImpl.createTaskId(id, "task-name", 0L);
+        submitJobAndStartTask(id, tid);
+
+        JobId id2 = new JobIdImpl(667L, "test-name");
+        TaskId tid2 = TaskIdImpl.createTaskId(id2, "task-name2", 0L);
+        submitJobAndStartTask(id2, tid2);
+
+        JobId id3 = new JobIdImpl(668L, "test-name");
+        TaskId tid3 = TaskIdImpl.createTaskId(id3, "task-name3", 0L);
+        submitJobAndStartTask(id3, tid3);
+
+        JobId id4 = new JobIdImpl(669L, "test-name");
+        TaskId tid4 = TaskIdImpl.createTaskId(id4, "task-name4", 0L);
+        submitJobAndStartTask(id4, tid4);
+
+        JobId id5 = new JobIdImpl(670L, "test-name");
+        TaskId tid5 = TaskIdImpl.createTaskId(id5, "task-name5", 0L);
+        submitJobAndStartTask(id5, tid5);
+
+        assertThat(liveJobs.getRunningTasks().size(), is(5));
+
+        assertThat(liveJobs.getRunningTask(tid).getTask().getName(), is("task-name"));
+        assertThat(liveJobs.getRunningTask(tid).getTask().getJobId(), is(id));
+
```

Comment:
you could also check that you can successfully get the task id and name of the four other tasks in this test